### PR TITLE
Convert more Java to Kotlin

### PR DESCRIPTION
**Pull request by hunterwerlla**
- Convert a decent number more files to Kotlin, try to clean up some conversion weirdness

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/schemas/code/DownloadCodeForSchemaPanel.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    lateinit var version: JComboBox<String>
+    lateinit var language: JComboBox<SchemaCodeLangs>
+    lateinit var location: TextFieldWithBrowseButton
+    private lateinit var versionModel: DefaultComboBoxModel<String>
+    private lateinit var languageModel: SortedComboBoxModel<SchemaCodeLangs>
+
+    private fun createUIComponents() {
+        versionModel = DefaultComboBoxModel()
+        version = ComboBox(versionModel)
+        languageModel = SortedComboBoxModel(Comparator.comparing(Function { obj: SchemaCodeLangs -> obj.toString() }, Comparator.naturalOrder()))
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/schemas/code/DownloadCodeForSchemaPanel.kt:33]: `SortedComboBoxModel(compareBy(Comparator.naturalOrder()) { it.toString() })`?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.kt [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+                    }
+                }
+            return
+        }
+        if (currentInput == saved) {
+            return
+        }
+        val path: Path
+        try {
+            path = Paths.get(currentInput)
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.kt:192]: split declaration is a bit weird, but i think the flow still makes more sense like this

**Review by rli — APPROVED** [review]

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/execution/PerContainerSettings.kt [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+import software.aws.toolkits.jetbrains.services.clouddebug.DebuggerSupport.Companion.debuggers
+import software.aws.toolkits.jetbrains.services.ecs.execution.DockerUtil.dockerPluginAvailable
+import software.aws.toolkits.jetbrains.ui.clouddebug.StartupCommandWithAutoFill
+import software.aws.toolkits.resources.message
+import java.awt.BorderLayout
+import javax.swing.JButton
+import javax.swing.JComponent
+import javax.swing.JPanel
+
+class PerContainerSettings internal constructor(private val project: Project, private val containerName: String, parent: Disposable) {
```
[the remark this answers is not in the document]
Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/execution/PerContainerSettings.kt:23]: Yeah, not really needed, removed

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/schemas/code/DownloadCodeForSchemaPanel.kt (hunk 1 above)
Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/schemas/code/DownloadCodeForSchemaPanel.kt:33]: I thought I caught all of these but aparently not, fixed

File: jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.kt (hunk 2 above)
Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.kt:192]: Yeah I tried refactoring it but decided to leave it as is

**Pull request merged**